flavors/vanilla/12/default.nix

```
@@ -106,7 +106,7 @@ in
       (pkgs.fetchurl {
```

```suggestion
      (pkgs.fetchpatch {
```

So that it doesn't happen again.